client/rest-high-level/src/main/java/org/elasticsearch/client/IndicesClient.java

```diff
@@ -58,6 +61,31 @@ public DeleteIndexResponse deleteIndex(DeleteIndexRequest deleteIndexRequest, He
      */
     public void deleteIndexAsync(DeleteIndexRequest deleteIndexRequest, ActionListener<DeleteIndexResponse> listener, Header... headers) {
         restHighLevelClient.performRequestAsyncAndParseEntity(deleteIndexRequest, Request::deleteIndex, DeleteIndexResponse::fromXContent,
-            listener, Collections.emptySet(), headers);
+            listener, emptySet(), headers);
+    }
+
+    public IndicesExistsResponse exists(IndicesExistsRequest indicesExistsRequest, Header... headers) throws IOException {
```

Comment:
can you make this final?

Reply:
I guess you mean the method and not parameters ("this" and not "these"), but I'd like to double check.: )

Reply:
yes the method.

Reply:
Done.